DOCS/man/options.rst

```diff
@@ -2544,6 +2544,15 @@ Subtitles
     offset scale factor, not what the video filter chain or the video output
     use.
 
+``--sub-vsfilter-bidi-compat=<yes|no>``
+    Set implicit bidi detection to ``ltr`` instead of ``auto`` to match ASS'
```

Comment:
```suggestion
    Set implicit bidi detection to ``ltr`` instead of ``auto`` to match ASS's
```
Should there be an `s`?

Reply:
Since the word ends with an `s` we don't need to. This is consistent with the manual in other places